nntrainer/src/neuralnet.cpp

```diff
@@ -575,14 +575,7 @@ void NeuralNetwork::backwarding(sharedConstTensor input,
     output = layers[i]->backwarding(output, iteration);
 }
 
-float NeuralNetwork::getLoss() {
-  loss = 0.0f;
-  for (unsigned int i = 0; i < layers.size(); i++) {
-    loss += layers[i]->getLoss();
-  }
-
-  return loss;
-}
+float NeuralNetwork::getLoss() { return layers.back()->getLoss(); }
```

Comment:
This will ignore the loss by the weight decay loss. Is this intentional?

Reply:
fixed.